app/controllers/transfers_controller.rb

```diff
@@ -21,19 +21,19 @@ def index
   def create
     @transfer = current_organization.transfers.new(transfer_params)
 
-    if @transfer.valid?
+    if !@transfer.valid? || @transfer.from_id == @transfer.to_id
```

Comment:
This feels backwards -- usually the affirmative case (`if true`) is the happy path and the negated case (`else`) is the error path.

If we can get this validation moved into the model instead then it can all fall under `@transfer.valid?`